build.yaml

```diff
@@ -113,43 +113,43 @@ steps:
      git rev-parse HEAD > git_version
    outputs:
      - from: /io/repo/auth/sql
-       to: /repo/auth/
+       to: /repo/auth/sql
      - from: /io/repo/ci/sql
-       to: /repo/ci/
+       to: /repo/ci/sql
      - from: /io/repo/batch/sql
-       to: /repo/batch/
+       to: /repo/batch/sql
      - from: /io/repo/batch/test
-       to: /repo/batch/
+       to: /repo/batch/test
      - from: /io/repo/memory/test
-       to: /repo/memory/
+       to: /repo/memory/test
      - from: /io/repo/query/test
-       to: /repo/query/
+       to: /repo/query/test
      - from: /io/repo/notebook/sql
-       to: /repo/notebook/
+       to: /repo/notebook/sql
      - from: /io/repo/monitoring/sql
-       to: /repo/monitoring/
+       to: /repo/monitoring/sql
      - from: /io/repo/atgu/sql
-       to: /repo/atgu/
+       to: /repo/atgu/sql
      - from: /io/repo/hail/python/hailtop
-       to: /repo/hailtop/
+       to: /repo/hailtop
      - from: /io/repo/hail/python/test
-       to: /repo/test/
+       to: /repo/test
      - from: /io/repo/hail/python/cluster-tests
-       to: /repo/hail/python/cluster-tests/
+       to: /repo/hail/python/cluster-tests
      - from: /io/repo/hail/python/hail/hail_version
        to: /hail_version
      - from: /io/repo/hail/python/hail/hail_pip_version
        to: /hail_pip_version
      - from: /io/repo/git_version
        to: /git_version
      - from: /io/repo/address/test
-       to: /repo/address/
+       to: /repo/address/test
      - from: /io/repo/lsm
        to: /repo/lsm
      - from: /io/repo/ci/bootstrap_create_accounts.py
-       to: /repo/ci/
+       to: /repo/ci/bootstrap_create_accounts.py
```

Comment:
Just to make sure. You have to be explicit here because batch always uses DEST_IS_TARGET.

Reply:
Correct!